### PR TITLE
chore: add aks-node-viewer

### PR DESCRIPTION
**Pull request by tallaxes**
**Description**

Adds aks-node-viewer to tooling

**How was this change tested?**

* `make presubmit` followed by `aks-node-viewer` (and `make az-node-viewer`)

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [x] No

**Release Note**

```release-note

```

### CODE REVIEW

File: hack/toolchain.sh [hunk 1]
```diff
@@ -25,6 +25,7 @@ tools() {
     go install github.com/mattn/goveralls@v0.0.12
     go install github.com/google/go-containerregistry/cmd/crane@v0.20.2
     go install github.com/go-swagger/go-swagger/cmd/swagger@v0.31.0
+    go install github.com/Azure/aks-node-viewer/cmd/aks-node-viewer@latest
```
Comment (Bryce-Soghigian) [inline hack/toolchain.sh:28]: should we update the az-node-viewer command too?

**Review by Bryce-Soghigian — DISMISSED** [review]

File: hack/toolchain.sh (hunk 1 above)
Comment (charliedmcb) [inline hack/toolchain.sh:28]: Should we be pinning to a hash here for dependency bot? Or does it not matter since it isn't directly in an action?

**Review by charliedmcb — APPROVED** [review]
One question, but looks good.

File: hack/toolchain.sh (hunk 1 above)
Comment (tallaxes) [reply hack/toolchain.sh:28]: Dependabot does not process these. It is up to us to decide, for just a dev tool that does not contribute to built code using latest makes sense to me.

Comment (charliedmcb) [reply hack/toolchain.sh:28]: Intuitively I feel like latest is best, so we don't have to continuously update this as we release changes, but curious the interplay between dependency bot and versioning within the repo.

Comment (charliedmcb) [reply hack/toolchain.sh:28]: Responded at the same time! Perfect. 100% agree since dependabot doesn't apply here.

**Pull request merged**